src/darwin/CHIPTool/CHIPTool/Framework Helpers/DefaultsUtils.m

```diff
@@ -65,30 +65,30 @@ void CHIPSetNextAvailableDeviceID(uint64_t id)
 @implementation CHIPToolPersistentStorageDelegate
 
 // MARK: CHIPPersistentStorageDelegate
-- (void)GetKeyValue:(NSString *)key handler:(SendKeyValue)completionHandler
+- (void)CHIPGetKeyValue:(NSString *)key handler:(SendKeyValue)completionHandler
```

Comment:
is C++ namespacing available to us if this file were a `.mm`?

Reply:
I think the issue is that the API exposed by the framework in `CHIPPersistentStorageDelegate.h` cannot use the namespace to disambiguate the function names. So, we need to add some prefix to each function. This file is implementing the protocol defined in `CHIPPersistentStorageDelegate.h`, so adding the namespace here won't solve the problem.